public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPrevious.tsx

```diff
@@ -14,11 +16,14 @@ export interface ReturnToPreviousProps {
 
 export const ReturnToPrevious = ({ href, title }: ReturnToPreviousProps) => {
   const styles = useStyles2(getStyles);
+  const { chrome } = useGrafana();
   const handleOnClick = () => {
-    console.log('Going to...', href);
+    setReturnToPrevious({ title: '', href: '' });
+    locationService.push(href);
```

Comment:
The 'button' should just be a link, so it functions as a link does it the browser (right click -> new tab, middle click to open in new tab, etc).